app/classes/Transvision/Strings.php

```diff
@@ -103,6 +103,32 @@ public static function multipleStringReplace($replacements, $string)
     }
 
     /**
+     * Highlight special characters in the string
+     *
+     * @param  string $string Source text
+     * @param  string $white_space Optional param to specify if we need to
```

Comment:
Should we use a boolean? It seems more logical (not sure if my first comment through you off here).

Not sure if you want to use $exclude_whitespaces (default True), to exclude changing too many existing calls.